main.go

```diff
@@ -19,6 +21,24 @@ func makeRedisKey(keys []string) string {
 	return strings.Join(keys, ":")
 }
 
+func calculateQueueLatency(contents string) float64 {
+	if contents == "" {
+		return 0
+	}
+
+	var job map[string]float64
+	if err := json.Unmarshal([]byte(contents), &job); err != nil {
+		return 0
+	}
```

Comment:
JSON パースに失敗した場合に何も表示されないとデバッグが大変そうなので、ここは一応エラーをログに出力しておくと良さそうです。

```suggestion
	if err := json.Unmarshal([]byte(contents), &job); err != nil {
		log.Println(err)
		return 0
	}
```

Reply:
f27845bffeb12877743493554aec69a6f4d0dab3 にてログ出力を追加しました 🔧